languages.json

```diff
@@ -450,14 +450,14 @@
                 ["\\\"", "\\\""],
                 ["'", "'"]
             ]
-        },
-        "Edn":{
-            "line_comment":[
-                ";"
-            ],
-            "extensions":[
-                "edn"
-            ]
+        },
+        "Edn":{
+            "line_comment":[
+                ";"
+            ],
+            "extensions":[
+                "edn"
+            ]

```

Comment:
I have no idea - I didn't notice because I have "ignore whitespace" turned on in GitKraken. I can try and work out what the actual change was and revert it.